### ISSUE TITLE
Allow to mock value in Gate.state using fork method 

### ISSUE THREAD

**Issue by ArthurSaenz** [effector/effector#505]
In tests of effector model need to find a way to mock value using this _code:_:

```ts
let scope = fork(root, {
  values: new Map()
    .set(Gate.state, {value: 'mock'}),
})
```

### 🎯 Expected Behavior

The `fork` can mock `Gate.state` like another store in. Expected the same behavior to independent store and Gate.state store

_For example:_ 

```ts
let scope = fork(root, {
  values: new Map()
    .set($curentFilters, '$curentFilters mock')   <-- working
    .set(Gate.state, {value: 'mock'}),            <-- not working
})
```

###  🔦 Current Behavior

The `fork` does not have any mock value `Gate.state` like an independent store.

### 🔍 Possible Solution

If it depends of babel configuration, could be better to see a reference or example of how to configure correctly for jest environment. 

### 📋 Steps to Reproduce

🔗 REPL: https://share.effector.dev/o0i6T2zt

1. Create Gate using the domain.
2. Try to create a test suite with a fork and try to mock Gate.state. For example, the most popular variant routing data.
3. Use `scope.getState(Gate.state)` for find value into Gate.state and getting not changed data. 
	- Also there are no exceptions and errors.

### 🗺️ Context (Environment)

**Usage packages:**
- effector (latest) 
- effector-react (latest)
- effector-root (latest) 
- jest (latest) 

### ℹ Detailed Description

The main use case is to create the test with the possibility mock Gate.state. Need an easy and intuitive way to mock Gate events and states. Maybe there is some advice or examples for operation with Gate in testing?

**Comment by sergeysova** [comment]
Reproduce: https://stackblitz.com/edit/fork-values-gate-state-error?file=src%2Fmain.tsx

```
Uncaught Error: unit should have a sid
```

**Comment by zerobias** [comment]
Should be fixed with #855
